### DISCUSSION TITLE
aws-lambd vs [@types/aws-lambda]

### DISCUSSION THREAD

**Discussion by mofach90** [DefinitelyTyped/DefinitelyTyped#68702] [Issues with a @types package] [answered]
I am new in Software Developement and i have a question regarding the usage of aws-lambd and @types/aws-lambda.

Do i still need to use the aws-lambd in my project even that @types/aws-lambda exist. I understood that @types/aws-lambda offers already the types definitions for typescript. or i missunderstood the concept ?

Thank you in advance

[a comment by an automated account is not shown]

**Reply by simonbuchan** [reply, marked as answer]
Yeah, you're right and you don't need the `aws-lambda` package, sorry about the confusion.

The `aws-lambda` package is unfortunately unrelated to this `@types/aws-lambda` package, so you don't need the former. Changing the name would unnecessarily inconvenience quite a few people though, so we put up with it for now.
